lib/validation-helpers/schemas.js

```diff
@@ -18,16 +18,25 @@ const customZipCodeValidation = function (value) {
   if (value.country === "SE") {
     value.zipCode = zipCode.replace(" ", ""); // Always save swedish zipcode without spacing.
   }
-  const {error: invalidZipCode} = joi
-    .string()
-    .postalCode(country ? country : "SE") // Default to SE
-    .validate(zipCode); // Validation doesn't care about whitespaces.
+
+  let invalidZipCode = isValidZipCode(zipCode, country);
   if (invalidZipCode) {
```

Comment:
Can't we just do the trim stuff right away? Less code, sure they might be run when not needed but seems like a small price to pay.

```suggestion
    const trimmedZipCode = zipCode.replace(/[a-zA-Z]+/g, ""); // Trim away letters
    const invalidZipCode = isValidZipCode(trimmedZipCode, country); // Validate again
    if (invalidZipCode) throw new Error(invalidZipCode.details[0].message);
    value.zipCode = trimmedZipCode;

```

Reply:
I guess not since some zipcodes always have letters in them, and then it would get invalid.